scripts/test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+process.env.NODE_ENV = 'test';
```

Comment:
It's not affecting anything in Jest itself but it is kind of a standard that people have adopted. For example, Relay's preprocessor behaves differently when this flag is set. Jest sets this in its binary (jest-cli/bin/jest.js) but since we are not it directly here we have to add it.